### ISSUE TITLE
SwallowedException should not flag exceptions named ignore*

### ISSUE THREAD

**Issue by vanniktech** [arturbosch/detekt#1408]
The following gets flagged:

```kotlin
} catch (ignored: AssertionError) {
```

Using RC12. Is there any way we can in the future take those considerations into account? It feels like it's the third rule that is missing this essential feature.
